lib/rules/no-extra-semicolons/index.js

```diff
@@ -87,6 +91,15 @@ const rule = function (actual) {
       const rawAfterNode = node.raws.after
 
       if (rawAfterNode && rawAfterNode.trim().length !== 0) {
+        /**
```

Comment:
As above, let's check if the last node is standard rule or a custom property set:

```js
        if (
          node.last
          && node.last.type === "rule"
          && !isCustomPropertySet(node.last)
          && !isStandardSyntaxRule(node.last)
        ) {
          return
        }
```